package.json

```diff
@@ -105,7 +105,7 @@
     "istanbul": "0.4.5",
     "mocha": "3.1.2",
     "should": "7.0.2",
-    "source-map-support": "^0.4.14",
+    "source-map-support": "^0.4.18",
```

Comment:
this should be strict version and maybe this should not be devDependency

Reply:
So moving this to dependencies should also resolve the previous comment right?